doc/api/net.md

```diff
@@ -618,8 +618,8 @@ For TCP connections, available `options` are:
 * `host` {string} Host the socket should connect to. **Default:** `'localhost'`.
 * `localAddress` {string} Local address the socket should connect from.
 * `localPort` {number} Local port the socket should connect from.
-* `family` {number}: Version of IP stack, can be either `4` or `6`.
-  **Default:** `4`.
+* `family` {number}: Version of IP stack. Must be `4`, `6`, or `0`.
```

Comment:
This may not be worth "fixing" as the fix may make the documentation more confusing while helping no one, but strictly speaking, if you pass a string value for `family`, it will work too. So both the `{number}` designation and the list of acceptable values are too narrow.

Reply:
(If that is worth fixing, it probably should go in a different pull request as it might result in discussion that this change as-is probably won't. Might as well get this in and then "fix" the above either later or never.)